app/src/main/java/co/jp/duonghb/MainActivity.kt

```diff
@@ -19,7 +19,7 @@ class MainActivity : ComponentActivity() {
             BitriseCIDemoTheme {
                 // A surface container using the 'background' color from the theme
                 Surface(
-                    modifier = Modifier.fillMaxSize(),
+                           modifier = Modifier.fillMaxSize(),
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/nokaduonghb/BitriseCIDemo/blob/23fbc44abd3f1aae6e499d5e6471855dd4c9c842/app/src/main/java/co/jp/duonghb/MainActivity.kt#L22"></span>Unexpected indentation (27) (should be 20)</td>
    </tr>
  </tbody>
</table>